modules/SitecoreImageBuilder/SitecoreImageBuilder.psm1

```diff
@@ -62,8 +98,13 @@ function Invoke-Build
         $spec = $_
         $sources = @()
 
-        $spec.Sources | ForEach-Object {
-            $sources += (Join-Path $InstallSourcePath $_.Name)
+        # Could be pulling remote resx now, so only grab for images
+        # we are intending to building
+        If ($spec.Include -and -not $Spec.Skip) {
```

Comment:
Where is `$spec.Skip` being set, can't find it anywhere?

Reply:
Think was was an early implementation before `$Exclude` or just future planning for the ability to deprecate image directly inside the `build.json`. Can remove/keep--your call.

Reply:
please remove